models/sfr/pincell/make_openmc_model.py

```diff
@@ -0,0 +1,42 @@
+import openmc
+from numpy import sqrt
+from argparse import ArgumentParser
+from models.sfr import common_input as pincell_params
+import models.sfr.openmc_pincells as pins
+from models.lwr.openmc_settings import COMMON_SETTINGS
+
+
+def argument_parser():
+    ap = ArgumentParser(description="SFR Pincell Model Generator")
+    ap.add_argument("-n", dest="n_axial", type=int, default=2, help="Number of cells in the Z direction")
+    ap.add_argument("-p", dest="pincell_type", type=str, choices=["inner", "outer"], default="inner",
+                    help="Material composition of the pincell fuel material")
+
+    return ap.parse_args()
+
+
+def generate_pincell_model(arguments):
+    pincell_universe, material = pins.PINCELLS[arguments.pincell_type]
+    fuel_bb = openmc.model.HexagonalPrism(edge_length=pincell_params.lattice_pitch / (sqrt(3)),
+                                          boundary_type="reflective", orientation="x")
+    top = openmc.ZPlane(z0=pincell_params.height, boundary_type="reflective")
+    bottom = openmc.ZPlane(z0=0, boundary_type="reflective")
```

Comment:
It makes the most sense to me to put these immediately before the definition of `pincell_universe_base` since they are only used there.